.gitignore

```diff
@@ -19,10 +19,10 @@ tools/MobBookUpdate/build/
 dist/
 
 #Config
-*.ini
+configuration.ini
```

Comment:
Added `configuration_template.ini`, so specified .gitignore to only ignore `configuration.ini`